tests/projects/spark-job/deployment/project-override-second.yml

```diff
@@ -1,6 +1,6 @@
 name: 'sparkJob-second'
 maintainer: ['MPyL']
-mpylVersion: 1.0.9
+mpylVersion: 1.7.0
```

Comment:
this field is for the upgrader script btw, it has nothing to do with the real mpyl version